packages/libraries/cli/src/base-command.ts

```diff
@@ -53,28 +53,19 @@ export default abstract class BaseCommand<T extends typeof Command> extends Comm
   }
 
   success(...args: any[]) {
-    this.log(colors.green('✔'), ...args);
+    this.log(Texture.success(...args));
   }
 
   fail(...args: any[]) {
-    this.log(colors.red('✖'), ...args);
+    this.log(Texture.failure(...args));
   }
 
   info(...args: any[]) {
-    this.log(colors.yellow('ℹ'), ...args);
+    this.log(Texture.info(...args));
   }
 
   infoWarning(...args: any[]) {
-    this.log(colors.yellow('⚠'), ...args);
-  }
-
-  bolderize(msg: string) {
```

Comment:
I like these functions being a higher level abstraction and agree with your decision to remove `bolderize` from here.